.github/workflows/eslint.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# ESLint is a tool for identifying and reporting on patterns
+# found in ECMAScript/JavaScript code.
+# More details at https://github.com/eslint/eslint
+# and https://eslint.org
+
+name: ESLint
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  eslint:
+    name: Run eslint scanning
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+          # We must fetch at least the immediate parents so that if this is
+          # a pull request then we can checkout the head.
+        with:
+          fetch-depth: 2
+
+      # If this run was triggered by a pull request event, then checkout
+      # the head of the pull request instead of the merge commit.
+      - run: git checkout HEAD^2
+        if: ${{ github.event_name == 'pull_request' }}
+
+      - name: Install ESLint
+        run: |
+          npm install eslint@8.57.0
+          npm install utf8
+          npm install jschardet
```

Comment:
there is no need to manually fix error with duplicated `suppressions` items, it's possible to use `SARIF_ESLINT_IGNORE_SUPPRESSED` env parameter to completely ignore suppressed warnings - https://github.com/FlowCrypt/flowcrypt-browser/blob/764daf24cbd7b855ef06888305dd76f45483f7a0/.github/workflows/eslint.yml#L45

Reply:
Thanks @sosnovsky - this definitely helps. By the way, may I know how do you come up having the "SARIF_ESLINT_IGNORE_SUPPRESSED" env parameter? This will help in my future engagements of similar cases. Thank you!

Reply:
I searched for `suppressed` in [eslint-formatter-sarif](https://github.com/microsoft/sarif-js-sdk/tree/main/packages/eslint-formatter-sarif) repo, and found recent commit which have added such functionality -  https://github.com/microsoft/sarif-js-sdk/commit/e8b2911611cdca34eb9b0f572ec6ef485b3bf10d

Reply:
Thanks, Roma - That was exceptional and very helpful.